File: 42.py
#题目：求一个3*3矩阵主对角线元素之和。
L=[[1,2,3],[4,5,6],[7,8,9]]
for i in L:
    print(i)
sum=0
for j in range(3):
    for k in range(3):
        x=L[j]
        y=x[k]
        if j==k:
            sum=sum+y
print("对角线之和为",sum)

File: C语言/42.cpp
//题目：求一个3*3矩阵主对角线元素之和。 
#include <stdio.h>
int main()
{
	int i,j,a[3][3];
	int sum=0;
	printf("请输入3*3的矩阵:");
	for(i=0;i<3;i++)
		for(j=0;j<3;j++)
		{
			scanf("%d",&a[i][j]);
			if(i==j) sum+=a[i][j]; 
		}	
	printf("主对角线之和为:%d",sum);
}
